.github/workflows/ci.yml

```
@@ -56,6 +75,7 @@ jobs:
         RAISE_ON_WARNING: 1
         MEASURE_COVERAGE: true
         RAILS_VERSION: ${{ matrix.rails_version }}
+        CAPTURE_PATCH_ENABLED: ${{ matrix.output_buffer == 'capture_patch_enabled' && 'true' || 'false' }}
```

Huh this looks... weird to me. Do we want `matrix.capture_patch_enabled` instead?

---

Yeah. With this logic, `CAPTURE_PATCH_ENABLED` is always getting set to `false`, even in the `capture_patch_enabled: true` variants of the CI matrix:
https://github.com/ViewComponent/view_component/actions/runs/4068065964/jobs/7006166117#step:5:10
which means those builds are actually skipping the tests they meant to run:
https://github.com/ViewComponent/view_component/actions/runs/4068065964/jobs/7006166117#step:5:568

With
```suggestion
        CAPTURE_PATCH_ENABLED: ${{ matrix.capture_patch_enabled }}
```
it runs as desired:
https://github.com/ViewComponent/view_component/actions/runs/4071219661/jobs/7012819664#step:5:10
https://github.com/ViewComponent/view_component/actions/runs/4071219661/jobs/7012819664#step:5:568

---

Yep! Just missed it on the migration over. I wasn't a fan of the build saying `true` or `false` so I'm changing up the approach to use a string value instead.